packages/format-library/src/index.js

```diff
@@ -8,18 +15,21 @@ import { italic } from './italic';
 import { link } from './link';
 import { strikethrough } from './strikethrough';
 
-/**
- * WordPress dependencies
- */
-import {
-	registerFormatType,
-} from '@wordpress/rich-text';
+export function getCoreFormatTypes() {
```

Comment:
JSDoc.

Reply:
Does it mean we are good with the proposal?